processors/base/helpers/__init__.py

```diff
@@ -128,7 +132,7 @@ def find_list_of_identifiers(text):
 
     # Pattern could be improved based on a extended
     # clinical trial identifiers format analysis
-    PATTERN = r'(%s[\d-]{3,})'
+    PATTERN = r'(%s[\s]*[\d-]{3,})'
```

Comment:
Minor minor thing that I wouldn't bother with usually, but as you'll remove the code from https://github.com/opentrials/processors/pull/67/files#r81792086, you can also simplify this a bit. There's no need to add the `[]` in `[\s]`, as it's only a single character (it's needed in `[\d-]` because there we want either `\d` or `-`), so you can rewrite this as `r'(%s\s*[\d-]{3,})'`